src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.js

```diff
@@ -44,6 +44,22 @@ const ImageRenderer = (props) => {
         Config.EXPENSIFY.URL_API_ROOT,
     );
 
+    const imageWidth = htmlAttribs['data-expensify-width'] ? parseInt(htmlAttribs['data-expensify-width'], 10) : undefined;
+    const imageHeight = htmlAttribs['data-expensify-height'] ? parseInt(htmlAttribs['data-expensify-height'], 10) : undefined;
+    const imagePreviewModalDisabled = htmlAttribs['data-expensify-preview-modal-disabled'] === 'true';
+
+    if (imagePreviewModalDisabled) {
```

Comment:
LGTM just one minor comment. Generally in our codebase we use a single return with either ternary or ampersand trees to display different components so can you update this such that the return below is like,
```
{imagePreviewModalDisabled ? (
    <ThumbnailImage
    ...
    >
) : (
    <AttachmentModal
   ...
   >
)}

Reply:
Fixed.

Reply:
⚡, i'll re-test.